src/pages/en/migrate.md

```diff
@@ -80,6 +80,18 @@ We love to find sensible defaults that "just work" out-of-the-box. As part of th
 
 Check our new [syntax highlighting docs](/en/guides/markdown-content/#syntax-highlighting) for full details. **If you prefer to keep Prism as your syntax highlighter,** [set the `syntaxHighlight` option to `'prism'`](/en/guides/markdown-content/#prism-configuration) in your project's markdown configuration.
 
+#### The `<Prism />` component has a new home
+
+As part of our mission to keep Astro core as lean as possible, we've moved the built-in `Prism` component out of `astro/components` and into the `@astrojs/prism` package. You can now import this component from `@astrojs/prism/component` like so:
+
+```astro
+---
+import Prism from '@astrojs/prism/component';
+---
+```
+
+Since the `@astrojs/prism` package is still bundled with `astro` core, you won't need to install anything new! However, note that we _do_ plan to extract `@astrojs/prism` (and Prism syntax highlighting in general) to a separate, installable package in the future. See [the `<Prism />` component API reference](/en/reference/api-reference#prism-) for more.
```

Comment:
```suggestion
Since the `@astrojs/prism` package is still bundled with `astro` core, you won't need to install anything new, nor add Prism as an integration! However, note that we _do_ plan to extract `@astrojs/prism` (and Prism syntax highlighting in general) to a separate, installable package in the future. See [the `<Prism />` component API reference](/en/reference/api-reference#prism-) for more.
```

(Just came to mind, to also help reinforce the new integrations-all-over-the-place mentality. Take it or leave it!)